cirq/work/observable_measurement_data.py

```diff
@@ -167,6 +176,18 @@ class BitstringAccumulator:
             arise. The total number of repetitions is the sum of this 1d array.
         timestamps: We record a timestamp for each request/chunk. This
             1d array will have the same length as `chunksizes`.
+        readout_calibration:
+            The result of `calibrate_readout_error`. When requesting
+            means and variances, if this is not None, we will use the
+            calibrated value to correct the requested quantity. This is a
+            BitstringAccumulator containing the results of measuring Z
+            observables with readout symmetrization enabled. This class
+            does *not* validate that both this parameter and the
+            BitstringAccumulator under construction contain measurements taken
+            with readout symmetrization turned on; and we strongly
+            encourage you to avoid initializing this directly. Instead, use
+            `measure_observables` and `calibrate_readout_error`.
```

Comment:
So, these don't exist yet, right? Is that the dependency on  #3647?

Reply:
removed; will add back in in #3647